### DISCUSSION TITLE
Set download directory of external page

### DISCUSSION THREAD

**Discussion by ThePoultryMan** [tauri-apps/tauri#10413] [Q&A]
If I have created a new window with the code below, can I set the download directory of files?
```rust
#[tauri::command]
pub async fn create_window(app_handle: tauri::AppHandle) {
    let window = tauri::WindowBuilder::new(
                &app_handle,
                "example_window",
                tauri::WindowUrl::External("https://example.com".parse().unwrap()),
            )
            .title("Example")
            .min_inner_size(1287.0, 256.0)
            .on_navigation(|url| {
                # handle navigation stuff here
    
                true
            })
            .build()
            .unwrap();
}
```
I know I can access the existing download directory with tauri's path api, but is there a way to set the directory? If not, can I get the location of where a file is downloaded to?

Bonus Question: Can I remove the popup that appears when a file is downloaded?

**Comment by FabianLars** [comment]
You're basically asking for the feature requested here https://github.com/tauri-apps/tauri/issues/4633 which is not implemented yet.
